browser/sources.gni

```diff
@@ -117,6 +116,7 @@ brave_chrome_browser_deps = [
   "//brave/components/brave_search/common:mojom",
   "//brave/components/brave_shields/common",
   "//brave/components/brave_sync/buildflags:buildflags",
+  "//brave/components/brave_talk/browser",
```

Comment:
I think we can delete this deps. Can't find any point that uses `brave_talk` targets from `brave/browser`

Reply:
Fixed